staff/views/users.py

```diff
@@ -39,9 +39,25 @@ def form_valid(self, form):
         return redirect(self.object.get_staff_url())
 
     def get_context_data(self, **kwargs):
+        orgs = [
+            {
+                "name": m.org.name,
+                "roles": [r.display_name for r in m.roles],
+                "staff_url": m.org.get_staff_url(),
+            }
+            for m in self.object.org_memberships.order_by("org__name")
+        ]
+        projects = [
+            {
+                "name": m.project.name,
+                "roles": [r.display_name for r in m.roles],
```

Comment:
Is this sorted?

Reply:
It's not, will fix this and the orgs one 👍

Reply:
Fixed in #954